lib/node_modules/@stdlib/repl/lib/ansi_colors.js

```diff
@@ -48,6 +48,33 @@ var ANSI = {
 	'brightCyan': '\u001b[96m',
 	'brightWhite': '\u001b[97m',
 
+	// Background colors:
+	'bgBlack': '\u001b[40m',
```

Comment:
@Snehil-Shah Are you including these so that a theme can color both the background and foreground text? Or for some other purpose?

Reply:
If someone wants they can add background colors to their tokens.